src/NServiceBus.Core.Tests/Persistence/InMemory/When_completing_a_saga_with_no_defined_unique_property.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.Core.Tests.Persistence.InMemory
+{
+    using System;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_unique_property
```

Comment:
This was written for the previous attempt at increasing the speed. It was necessary there to ensure that all of the 'cleanup' operations performed against different collections of internal data during `Complete` were done without error. I'm not so sure that it's needed with this new implementation